### ISSUE TITLE
test_malaria takes too long

### ISSUE THREAD

**Issue by JihongJu** [broadinstitute/keras-rcnn#29]
Maybe we could mock the downloading procedure?

**Comment by 0x00b1** [comment]
@JihongJu I agree. It’ll also save us some money. Especially when providing COCO and VOC (I’m working on a VOC dataset this weekend).

**Comment by jhung0** [comment]
@JihongJu what do you mean by mock the downloading procedure?

**Comment by JihongJu** [comment]
So if you have operations that you don't want them to be executed during unit testing, you can "mock" these operations so that they will be replaced by "fake" ones. You can find a better summary of "mocking" [here](https://stackoverflow.com/questions/2665812/what-is-mocking), and a library for Python [here](https://docs.python.org/3/library/unittest.mock.html).
